code/_core/obj/item/fishing/rod.dm

```diff
@@ -44,6 +44,13 @@
 	LOADATOM("line")
 	LOADATOM("bait")
 	
+/obj/item/fishing/rod/update_value()
```

Comment:
This should be ``get_value()`` instead. Check out ``/obj/item/weapon/ranged/energy/get_value()`` on how that should work.